webui/tests/bin/createUserAndExperiments.py

```diff
@@ -56,17 +61,21 @@ def create_user(username, password=""):
 login_as(USER_WITH_PASSWORD_USERNAME, USER_WITH_PASSWORD_PASSWORD)
 
 # Create experiments
-determined_root_dir = pathlib.Path(__file__).absolute().parents[3]
-experiment_dir = determined_root_dir.joinpath("e2e_tests", "tests", "fixtures", "no_op")
-
-for _ in range(4):
-    subprocess.run(
+def createExperiment(directory, config_file, count):
+    cmd = " ".join(
         [
             "det",
             "experiment",
             "create",
-            str(experiment_dir.joinpath("single-very-many-long-steps.yaml")),
-            str(experiment_dir),
-        ],
-        check=True,
+            str(directory.joinpath(config_file)),
+            str(directory),
+        ]
     )
+
+    return subprocess.run(
+        f"seq {count} | xargs -n 1 -P 0 -I[] {cmd}", shell=True, check=True,
```

Comment:
can you test that this works on mac?

Reply:
@hamidzr For this part I encounter this error:
```
no matches found: -I[]
```
It didn't like the -I[] option...

Reply:
swapped it for a python solution.